libmachine/host.go

```diff
@@ -379,9 +381,17 @@ func getHostState(host Host, hostListItemsChan chan<- HostListItem) {
 
 	dockerHost := os.Getenv("DOCKER_HOST")
 
+	normalActive := dockerHost == url && currentState != state.Stopped
+
+	replacedSwarmPort := strings.Replace(url, ":2376", ":3376", -1)
+
+	// TODO: HUGE HORRIBLE HACK!!!! DO NOT HARDCODE DOCKER PORT 2376 OR SWARM PORT 3376
```

Comment:
Yeah, I think I will be OK with putting all this stuff in with some tests and maybe more comments, otherwise I worry a lot of fragility and hard-to-understandishness of this solution, longer-termish solution would definitely be to query for the swarm and docker ports / addresses and test those instead.